packages/components/psammead-section-divider/src/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import { boolean, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import {
+  getDoublePica,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
+import { C_CLOUD_LIGHT, C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+
+const TOP_MARGIN_PX = 1;
+
+const calcTop = ({ inline, script }) =>
+  inline
+    ? `
+      // place at middle of text line height
+      top: ${(script.doublePica.groupA.lineHeight / 2 + TOP_MARGIN_PX) / 16}rem;
+
+      ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+        top: ${(script.doublePica.groupB.lineHeight / 2 + TOP_MARGIN_PX) /
+          16}rem;
+      }
+
+      ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+        top: ${(script.doublePica.groupD.lineHeight / 2 + TOP_MARGIN_PX) /
+          16}rem;
+      }
+    `
+    : 'top: 0;';
+
+const SectionDividerWrapper = styled.div`
+  position: relative;
+  &::before {
+    content: '';
+    position: absolute;
+    height: 1px;
+    border: none;
+    background: ${C_CLOUD_LIGHT};
+    left: 0;
+    right: 0;
+    ${calcTop};
+  }
+`;
+
+const SectionTitle = styled.h2`
+  ${props => getDoublePica(props.script)};
+  color: ${C_EBON};
+  background-color: ${C_WHITE};
+  font-family: ${GEL_FF_REITH_SANS};
+  display: inline-block;
+  position: relative;
+  margin: ${TOP_MARGIN_PX}px 0 0 0;
+
+  html:not([dir='rtl']) & {
```

Comment:
Whats the browser support for this like? Im not sure if I prefer this, vs passing `dir` as a prop to the component

Reply:
If we do, We can probably reduce the complexity here if we just account for the `html[dir='rtl']`

Have it default to `padding: 0 ${GEL_SPACING} 0 0;` then override if `html[dir='rtl']` to `padding: 0 0 0 ${GEL_SPACING};`

Reply:
chatting to @pjlee11 in #si_repo_psammead on slack, we should pass `dir` as a prop and toggle on `rtl` as a value

Reply:
Yep, that sounds good. I don't have any attachment to this method, I was just trying to make the rtl toggle on storybook work (which may need some thought about how that would work with dir as a prop)